KS-5-1 задача 1

VVShchegolev opened the pull request.

Commit:
KS-5-1 задача 1

ievetrovdev commented on src/main/kotlin/Lesson_5_Ex_1/Lesson_5_Ex_1.kt:
```diff
@@ -0,0 +1,17 @@
+
+}
+
+const val NUMBER1 = 5
```
Захардкоженные значения NUMBER1 и NUMBER2 делают пример всегда одинаковым. Лучше генерировать случайные числа, чтобы капча была динамической и не теряла смысл. Посмотри в сторону использования экстеншена получения рандомного числа из диапазона .random()

ievetrovdev commented on src/main/kotlin/Lesson_5_Ex_1/Lesson_5_Ex_1.kt:
```diff
@@ -0,0 +1,17 @@
+
+const val NUMBER1 = 5
+const val NUMBER2 = 3
+const val CORRECT_SUM = NUMBER1 + NUMBER2
```
Можно вычислить сумму прямо в условии, создание аж константы для такой простой задачи несколько избыточно)

VVShchegolev committed:
Update KS_5_1 Задача 1 исправлена

Что исправлено, удалены константы и заменил захардженные значения на случайные числа из диапазона.random().

ievetrovdev commented on src/main/kotlin/Lesson_5_Ex_1/Lesson_5_Ex_1.kt:
```diff
@@ -0,0 +1,16 @@
+package Lesson_5_Ex_1
+
+
```
Двойные пустые строки размывают внимание, одной строки всегда достаточно)

ievetrovdev commented on src/main/kotlin/Lesson_5_Ex_1/Lesson_5_Ex_1.kt:
```diff
@@ -0,0 +1,16 @@
+
+    val userAnswer = readln().toInt()
+
+    println(if (userAnswer == number1 + number2) "Добро пожаловать!" else "Доступ запрещен.")
```
Для читаемости лучше добавить форматирование, перенеся условие на новую строку, примерно так:
```kotlin
    println(
        if (userAnswer == number1 + number2) "Добро пожаловать!"
        else "Доступ запрещен."
    )

ievetrovdev reviewed (approved):
Отлично!
